#--- a/third-party/fb-mysql/CMakeLists.txt	2022-06-16 16:50:22.283000000 +0200
#+++ b/third-party/fb-mysql/CMakeLists.txt	2022-06-16 18:30:00.373739502 +0200
@@ -12,10 +12,6 @@

 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   FB_MYSQL_DOWNLOAD_ARGS
-  SOURCE_URL
-  "https://github.com/facebook/mysql-5.6/archive/refs/tags/fb-prod8-202101.tar.gz"
-  SOURCE_HASH
-  "SHA512=4e07ae4e6628792ec5d77af7e524bddc2e9ac361dff4b93060f9fb5804d72a7144824ac84138487a3b4dcac350453cd5f17afd9a951b9d8248c292bf378e1e78"
 )
 ExternalProject_Add(
   bundled_fbmysqlclient
--- a/third-party/proxygen/CMakeLists.txt	2022-06-16 16:51:48.652000000 +0200
+++ b/third-party/proxygen/CMakeLists.txt	2022-06-16 18:30:00.391586726 +0200
@@ -18,10 +18,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   PROXYGEN_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/facebook/proxygen/releases/download/v2022.01.31.00/proxygen-v2022.01.31.00.tar.gz"
-  SOURCE_HASH
-  "SHA256=5360a8ccdfb2f5a6c7b3eed331ec7ab0e2c792d579c6fff499c85c516c11fe14"
 )
 
 ExternalProject_Add(
--- a/third-party/wangle/CMakeLists.txt	2022-06-16 16:51:48.652000000 +0200
+++ b/third-party/wangle/CMakeLists.txt	2022-06-16 18:30:00.394195460 +0200
@@ -20,10 +20,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   WANGLE_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/facebook/wangle/releases/download/v2022.01.31.00/wangle-v2022.01.31.00.tar.gz"
-  SOURCE_HASH
-  "SHA256=1002e9c32b6f4837f6a760016e3b3e22f3509880ef3eaad191c80dc92655f23f"
 )
 
 ExternalProject_Add(
--- a/third-party/squangle/CMakeLists.txt	2022-06-16 16:51:48.652000000 +0200
+++ b/third-party/squangle/CMakeLists.txt	2022-06-16 18:30:00.396715727 +0200
@@ -16,10 +16,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   SQUANGLE_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/facebook/squangle/archive/refs/tags/v2022.01.31.00.tar.gz"
-  SOURCE_HASH
-  "SHA256=78988eacf99d380da4c660161bcb20305e4b54369e17b1bb866c5fb188acff76"
   FILENAME_PREFIX
   "squangle-"
 )
--- a/third-party/watchman/CMakeLists.txt	2022-06-16 16:51:48.652000000 +0200
+++ b/third-party/watchman/CMakeLists.txt	2022-06-16 18:30:00.404187081 +0200
@@ -5,10 +5,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   WATCHMAN_DOWNLOAD_ARGS
-  SOURCE_URL "https://github.com/facebook/watchman/archive/refs/tags/v2022.01.31.00.tar.gz"
-  SOURCE_HASH "SHA256=5a253c289141d19b8c6fb05e4d12a75343c62d236f98dbbf6af4a50dc0550d90"
-  # The tarball name is just the tag name, which can conflict in the cache
-  FILENAME_PREFIX "watchman-"
 )
 
 get_target_property(BOOST_INCLUDE_DIR boost INTERFACE_INCLUDE_DIRECTORIES)
--- a/third-party/thrift/CMakeLists.txt	2022-06-16 16:51:48.652000000 +0200
+++ b/third-party/thrift/CMakeLists.txt	2022-06-16 18:30:00.406737187 +0200
@@ -20,10 +20,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   THRIFT_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/facebook/fbthrift/archive/refs/tags/v2022.01.31.00.tar.gz"
-  SOURCE_HASH
-  "SHA256=6194127fd9e6771bd34f502a84b292278bf3a6ee7b87377afd1ae287a5572f48"
   FILENAME_PREFIX
   "fbthrift-"
 )
--- a/third-party/mcrouter/CMakeLists.txt	2022-06-16 16:51:48.651000000 +0200
+++ b/third-party/mcrouter/CMakeLists.txt	2022-06-16 18:30:00.412827594 +0200
@@ -5,10 +5,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   MCROUTER_DOWNLOAD_ARGS
-  SOURCE_URL "https://github.com/facebook/mcrouter/archive/refs/tags/v2022.01.31.00.tar.gz"
-  SOURCE_HASH "SHA256=478b8d0b88bdca7c65863764b50dc46f92d849c39f8b34ecc657884106c9b4e6"
-  # The tarball name is just the tag name, which can conflict in the cache
-  FILENAME_PREFIX "mcrouter-"
 )
 
 set(
--- a/third-party/timelib/CMakeLists.txt	2022-06-16 16:51:48.652000000 +0200
+++ b/third-party/timelib/CMakeLists.txt	2022-06-16 18:30:00.417958866 +0200
@@ -6,10 +6,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   TIMELIB_DOWNLOAD_ARGS
-  SOURCE_URL "https://github.com/derickr/timelib/archive/refs/tags/2021.07.tar.gz"
-  SOURCE_HASH "SHA512=7bc56d20360937af10f63960e443cc8bd4d24c5369f697241e54da21465d4512bd16cfa6f0efcf2b847cc19781e1cecf93c9e19a1efa4f1a7012c9fa442eeabe"
-  # Look for timelib-YYYY.MM.tar.gz in download cache, not just YYYY.MM.tar.gz
-  FILENAME_PREFIX "timelib-"
 )
 
 if(USE_BUNDLED_TZDATA)
--- a/third-party/folly/CMakeLists.txt	2022-06-16 16:51:48.651000000 +0200
+++ b/third-party/folly/CMakeLists.txt	2022-06-16 18:30:00.433286243 +0200
@@ -3,10 +3,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   FOLLY_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/facebook/folly/releases/download/v2022.01.31.00/folly-v2022.01.31.00.tar.gz"
-  SOURCE_HASH
-  "SHA256=7b8d5dd2eb51757858247af0ad27af2e3e93823f84033a628722b01e06cd68a9"
 )
 
 get_target_property(BOOST_INCLUDE_DIR boost INTERFACE_INCLUDE_DIRECTORIES)
--- a/third-party/rustc/CMakeLists.txt	2022-06-17 13:07:47.767000000 +0200
+++ b/third-party/rustc/CMakeLists.txt	2022-06-17 13:07:58.963000000 +0200
@@ -15,14 +15,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   RUST_DOWNLOAD_ARGS
-  Linux_URL
-  "https://static.rust-lang.org/dist/${RUST_NIGHTLY_VERSION}/rust-nightly-x86_64-unknown-linux-gnu.tar.gz"
-  Darwin_URL
-  "https://static.rust-lang.org/dist/${RUST_NIGHTLY_VERSION}/rust-nightly-x86_64-apple-darwin.tar.gz"
-  Linux_HASH
-  "SHA512=a7ec879851bd1bb8bf57b77860c4d99b0b76702182782520ab5f5053b6b113f3a0890c1e8210f8c010f43ea0804abdeeb90422534498e47178cad3643ddcca32"
-  Darwin_HASH
-  "SHA512=b5655f92605e9a69b3ce49e73b92a960c4bc9bcb34edcef0d6b4829cec7258b0d00f398f41b0ce1fc399fe3cc35c91ca347eb2e0fe6b853aa94167f6dcf58697"
   # The original filename doesn't contain any version information, so add the version information as a prefix to avoid cache collisions when updating later
   FILENAME_PREFIX "rustc-${RUST_NIGHTLY_VERSION}-"
 )
--- a/third-party/re2/CMakeLists.txt	2022-06-17 15:01:26.146000000 +0200
+++ b/third-party/re2/CMakeLists.txt	2022-06-17 15:02:34.110000000 +0200
@@ -12,10 +12,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   RE2_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/google/re2/archive/refs/tags/2021-11-01.tar.gz"
-  SOURCE_HASH
-  "SHA256=8c45f7fba029ab41f2a7e6545058d9eec94eef97ce70df58e92d85cfc08b4669"
   FILENAME_PREFIX "re2-"
 )

--- a/third-party/brotli/CMakeLists.txt	2022-06-17 14:02:02.526000000 +0200
+++ b/third-party/brotli/CMakeLists.txt	2022-06-17 14:02:09.824000000 +0200
@@ -4,10 +4,6 @@
 include(HPHPFunctions)
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   BROTLI_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/google/brotli/archive/refs/tags/v1.0.9.tar.gz"
-  SOURCE_HASH
-  "SHA256=f9e8d81d0405ba66d181529af42a3354f838c939095ff99930da6aa9cdf6fe46"
   FILENAME_PREFIX "brotli-"
 )
 
--- a/third-party/fmt/CMakeLists.txt	2022-06-18 16:55:18.677000000 +0200
+++ b/third-party/fmt/CMakeLists.txt	2022-06-18 16:55:29.015000000 +0200
@@ -3,10 +3,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   FMT_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/fmtlib/fmt/releases/download/6.1.2/fmt-6.1.2.zip"
-  SOURCE_HASH
-  "SHA512=d21085a2010786ff18c47acb033d9e4d51a3d58f9707cd9adf0f44642c1e4d80fd8cddafe58d95bb4f3e4a84ac5799caafead4a9feb12cc549b03d4d389fcc93"
 )
 
 set(INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/fmt-prefix")
--- a/third-party/fizz/CMakeLists.txt	2022-06-19 10:24:50.285000000 +0200
+++ b/third-party/fizz/CMakeLists.txt	2022-06-19 10:25:03.587000000 +0200
@@ -22,10 +22,6 @@
 
 SET_HHVM_THIRD_PARTY_SOURCE_ARGS(
   FIZZ_SOURCE_ARGS
-  SOURCE_URL
-  "https://github.com/facebookincubator/fizz/releases/download/v2022.01.31.00/fizz-v2022.01.31.00.tar.gz"
-  SOURCE_HASH
-  "SHA256=32a60e78d41ea2682ce7e5d741b964f0ea83642656e42d4fea90c0936d6d0c7d"
 )
 
 set(
